tests/test_operations.py

```diff
@@ -36,9 +36,10 @@ def mocked_env(mocker):
     mock_env.blueprint_dir = BLUEPRINT_DIR
     mock_env.plugin_manager = PLUGIN_MANAGER
     mock_env.service_template_name = SERVICE_TEMPLATE_NAME
-    mock_env.model_storage.service_template.list = mocker.MagicMock(
-        return_value=[]
-    )
+    # Each create execution checks that there is not pre-existing service
```

Comment:
that there no existing service templates with the same name as the current service template.